### PR TITLE
QUEST_JOBSTEP.tsv Review Lines Part 2

### PR DESCRIPTION
**Pull request by ttgmichael**
Reviewing lines for Jobstep to check for outdated terms used, grammar problems, typos, and weird item references. 

Let me know if you catch any small things within these changes.

### CODE REVIEW

File: QUEST_JOBSTEP.tsv [hunk 1]
```diff
@@ -1281,8 +1281,8 @@ QUEST_JOBSTEP_20150317_001280	{memo X}Hand over the Merrogg Leather to the Quarr
 QUEST_JOBSTEP_20150317_001281	{memo X}You've obtained enough Merrogg Leather that are needed by the Quarrel Shooter Master. Hand over the Merrogg Leather to the Quarrel Shooter Master in Klaipeda.
 QUEST_JOBSTEP_20150317_001282	{memo X}Obtain %s by defeating Merogs
 QUEST_JOBSTEP_20150317_001283	The Hunter Master doesn't express it much but he seems happy to have found the book. Talk to him about the next book to find.
-QUEST_JOBSTEP_20150317_001284	Collect rocks from the Crystal Mine entrance in Miner's Village
-QUEST_JOBSTEP_20150317_001285	Archer master has the next book. Collect rocks from the Crystal Mine entrance in Miner's Village for the Archer Master.
+QUEST_JOBSTEP_20150317_001284	Collect rocks from the Crystal Mine entrance in Miners' Village
+QUEST_JOBSTEP_20150317_001285	Archer Master has the next book. Collect rocks from the Crystal Mine entrance in Miners' Village for the Archer Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Archer Master -> The Archer Master

File: QUEST_JOBSTEP.tsv [hunk 2]
```diff
@@ -1301,7 +1301,7 @@ QUEST_JOBSTEP_20150317_001300	The Hunter Master has the book the Quarrel Shooter
 QUEST_JOBSTEP_20150317_001301	Give silver to the Hunter Master
 QUEST_JOBSTEP_20150317_001302	Prepared enough money for the book. Give 5000 silver to the Hunter Master in exchange for the book and take the book to the Quarrel Shooter Master.
 QUEST_JOBSTEP_20150317_001303	The Quarrel Shooter Master seems utterly happy to get the book. Talk to him to find the next book.
-QUEST_JOBSTEP_20150317_001304	Collect rocks from Miner's Village
+QUEST_JOBSTEP_20150317_001304	Collect rocks from Miners' Village
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: from -> in/at?

File: QUEST_JOBSTEP.tsv [hunk 3]
```diff
@@ -1941,26 +1941,26 @@ QUEST_JOBSTEP_20150317_001940	Report to the Bokor Master
...
 QUEST_JOBSTEP_20150317_001954	Helped the Bokor Master's research. Report to the Pyromancer Master.
 QUEST_JOBSTEP_20150317_001955	{memo X}Summoned Evil
-QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer master in Klaipeda.
-QUEST_JOBSTEP_20150317_001957	Open the box of the Pyromancer Mastert
-QUEST_JOBSTEP_20150317_001958	The Cryomancer Master told you to open the box in the Pyromancer Master's room as a condition to accept you. Something about it feels uncomfortable but open the box anyway.
-QUEST_JOBSTEP_20150317_001959	Report to Cryomancer master
+QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer Master in Klaipeda.
+QUEST_JOBSTEP_20150317_001957	Open the Pyromancer Master's Box
+QUEST_JOBSTEP_20150317_001958	The Cryomancer Master told you to open a box in the Pyromancer Master's room as a condition to accept you. Something about it feels uncomfortable but open the box anyway.
+QUEST_JOBSTEP_20150317_001959	Report to Cryomancer Master
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: to the ...

File: QUEST_JOBSTEP.tsv [hunk 4]
```diff
@@ -1941,26 +1941,26 @@ QUEST_JOBSTEP_20150317_001940	Report to the Bokor Master
...
-QUEST_JOBSTEP_20150317_001953	The Pyromancer Master says he'll accept you if you help with his research. Go to the Bokor Master who is working on the research together and help her study.
+QUEST_JOBSTEP_20150317_001952	Help with the Bokor Master's research
+QUEST_JOBSTEP_20150317_001953	The Pyromancer Master says he'll accept you if you can help with his research. Go to the Bokor Master who is collaborating and help her with her research.
 QUEST_JOBSTEP_20150317_001954	Helped the Bokor Master's research. Report to the Pyromancer Master.
 QUEST_JOBSTEP_20150317_001955	{memo X}Summoned Evil
-QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer master in Klaipeda.
-QUEST_JOBSTEP_20150317_001957	Open the box of the Pyromancer Mastert
-QUEST_JOBSTEP_20150317_001958	The Cryomancer Master told you to open the box in the Pyromancer Master's room as a condition to accept you. Something about it feels uncomfortable but open the box anyway.
-QUEST_JOBSTEP_20150317_001959	Report to Cryomancer master
+QUEST_JOBSTEP_20150317_001956	Talk to Cryomancer Master in Klaipeda.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: talk to the ..

File: QUEST_JOBSTEP.tsv [hunk 5]
```diff
@@ -2822,29 +2822,29 @@ QUEST_JOBSTEP_20150714_002821	{memo X}The Peltasta Master handed over a mission
...
-QUEST_JOBSTEP_20150714_002831	The Sapper Master asked you to help his experiment. First, collect the Burnable Juice of Tontulias from Zachariel Crossroads.
-QUEST_JOBSTEP_20150714_002832	The Pyromancer Master requested to you something in return for teaching high level magic. Put the Pot of Sun on Ramstis Ridge.
-QUEST_JOBSTEP_20150714_002833	Collect hard barks of Varvs and rigid horns of Zinutekas which are the ingredients Cryomancer wants from the 1st floor of Royal Mausoleum.
+QUEST_JOBSTEP_20150714_002828	Get Glizardon Horns from Tenet Chapel Basement 1F, Galok Feathers from 1F, {nl}and Inoum Fragments from 2F that the Wizard Master wants.
+QUEST_JOBSTEP_20150714_002829	The Cleric Master asked you to help curing the patients. Obtain the Holy Water from the Priest Master and the Krivis Master and cure the patients at Miners' Village.
+QUEST_JOBSTEP_20150714_002830	The Psychokino Master told you that he will help you to spurt your real abilities, but he wants demonic essences as a compensation. Defeat Minotaur and Chapparitions and obtain demonic essences.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Chapparitions -> Chapparition

File: QUEST_JOBSTEP.tsv [hunk 6]
```diff
@@ -2822,29 +2822,29 @@ QUEST_JOBSTEP_20150714_002821	{memo X}The Peltasta Master handed over a mission
...
-QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up 4 levels.
-QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced 4 levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
-QUEST_JOBSTEP_20150714_002839	Successfully enhanced shield type item to 4 levels. Report to the Rodelero Master.
+QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up four levels.
+QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced four levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
+QUEST_JOBSTEP_20150714_002839	Successfully enhanced shield type item to four levels. Report to the Rodelero Master.
 QUEST_JOBSTEP_20150714_002840	Talk to the Thaumaturge Master in Gateway to Great King.
-QUEST_JOBSTEP_20150714_002841	Obtain resins of Varvs, tail fragments of Zinutekas and activating nuclues of Boowookes from the 1st floor of Royal Royal Mausoleum.
+QUEST_JOBSTEP_20150714_002841	Obtain Varb Resin, Zinutekases Tail Fragments, and Boowook Activation Cores from the 1st floor of Royal Mausoleum.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Zinutekases -> Zinutekas

File: QUEST_JOBSTEP.tsv [hunk 7]
```diff
@@ -2822,29 +2822,29 @@ QUEST_JOBSTEP_20150714_002821	{memo X}The Peltasta Master handed over a mission
...
 QUEST_JOBSTEP_20150714_002834	The Sadhu Master wants to know whether you are ready to accept the training. Duel against the Sadhu Master and win.
 QUEST_JOBSTEP_20150714_002835	Talk to the Wugushi Master in Gateway of Great King
 QUEST_JOBSTEP_20150714_002836	The Wugushi Master says that even if cowardly, the last one standing is the true winner of the combat. Have a duel with the Wugushi Master.
-QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up 4 levels.
-QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced 4 levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
-QUEST_JOBSTEP_20150714_002839	Successfully enhanced shield type item to 4 levels. Report to the Rodelero Master.
+QUEST_JOBSTEP_20150714_002837	Enhance Shield type item up four levels.
+QUEST_JOBSTEP_20150714_002838	The Rodelero Master asked you to being shield that is enhanced four levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: The Rodelero Master asked you to being shield that is enhanced four levels. Enhance shield type item and bring it to the Rodelero Master to earn his recognition. 

to

The Rodelero Master asked you to bring your shield with an enhance level of 4. Enhance a shield-type item and bring it to the Rodelero Master to earn his recognition.

File: QUEST_JOBSTEP.tsv [hunk 8]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
-QUEST_JOBSTEP_20150714_002859	The Squire Master asked you to practice Arrest rope as much as you can so you can control it without any problem. Practice to use the rope on three mosnter at once in Poslinkis Forest. 
+QUEST_JOBSTEP_20150714_002859	The Squire Master asked you to practice Arrest rope as much as you can so you can control it without any problem. Practice to use the rope on three monster at once in Poslinkis Forest. 
 QUEST_JOBSTEP_20150714_002860	Go to the Centurion Master in Forest of Prayer.
 QUEST_JOBSTEP_20150714_002861	The Wugushi Master is waiting for you in the Gateway of Great King.
 QUEST_JOBSTEP_20150714_002862	The Scout Master is waiting for you in the Gateway of Great King.
-QUEST_JOBSTEP_20150714_002863	The Scout Master wants to check your abilities. Defeat Deadvones at the 1st district of Demons' Prison and return to the Scout Master.
+QUEST_JOBSTEP_20150714_002863	The Scout Master wants to check your abilities. Defeat Deadvones at District 1 of Demons' Prison and return to the Scout Master.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Defeat Deadvones -> Defeat Deadborn

File: QUEST_JOBSTEP.tsv [hunk 9]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
 QUEST_JOBSTEP_20150714_002864	The Fletcher Master is waiting for you in Goddess' Ancient Garden.
-QUEST_JOBSTEP_20150714_002865	The Fletcher Master told you that if you could bring him well burnable firestones, he would teach you better skills, but you can't really trust him. Defeat Yonazolems in Aqueduct Bridge Area and collect well burnable firestones.
+QUEST_JOBSTEP_20150714_002865	The Fletcher Master told you that if you could bring him Well-Burnable Firestones, he would teach you better skills, but you can't really trust him. Defeat Yonazolems in Aqueduct Bridge Area and collect Well-Burnable Firestones.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Yonazolems -> Yonazolem

Well-Burnable Firestones should be Well-Burnable Firestone then.

File: QUEST_JOBSTEP.tsv [hunk 10]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
 QUEST_JOBSTEP_20150714_002866	Schwarzer Reiter is waiting for you in Forest of Prayer.
 QUEST_JOBSTEP_20150714_002867	Go to the Monk Master in Goddess' Ancient Garden.
-QUEST_JOBSTEP_20150714_002868	Collect sharp Canines from Netherbovine at Poslinkis Forest
-QUEST_JOBSTEP_20150714_002869	The Monk Master asked you to bring the proof of the determination. Defeat Netherbovine at Poslinkis Forest and bring sharp Canines as a proof.
+QUEST_JOBSTEP_20150714_002868	Collect Sharp Canines from Netherbovine at Poslinkis Forest
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Sharp Canines -> a Sharp Canine

File: QUEST_JOBSTEP.tsv [hunk 11]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
+QUEST_JOBSTEP_20150714_002869	The Monk Master asked you to bring the proof of the determination. Defeat Netherbovine at Poslinkis Forest and bring Sharp Canines as a proof.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Sharp Canines -> a Sharp Canine

File: QUEST_JOBSTEP.tsv [hunk 12]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
 QUEST_JOBSTEP_20150714_002874	Talk to the Monk Master in the Goddess' Ancient Garden.
-QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants hardened oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
-QUEST_JOBSTEP_20150714_002876	go to the Sorcerer Master in Forest of Prayer.
-QUEST_JOBSTEP_20150714_002877	Rodolero Master told you that you are now a true Rodelero, but also told you that you can still improve. Defeat Colimencia at the garden of Green Family and find out what's lacking in you.
+QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants Hardened Oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
+QUEST_JOBSTEP_20150714_002876	Go to the Sorcerer Master in Forest of Prayer.
+QUEST_JOBSTEP_20150714_002877	Rodolero Master told you that you are now a true Rodelero, but also told you that you can still improve. Defeat Colimencia at the Greene Manor and find out what's lacking in you.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Rodolero Master -> The Rodelero Master

File: QUEST_JOBSTEP.tsv [hunk 13]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
 QUEST_JOBSTEP_20150714_002878	Go to the Cataphract Master in Gateway of Great King.
 QUEST_JOBSTEP_20150714_002879	Go to the Thaumaturge Master in Gateway of Great King.
 QUEST_JOBSTEP_20150714_002880	The Thaumaturge Master wants to know whether you would be able to surpass the next level. Defeat Netherbovine that is making a mess in Poslinkis Forest.
 QUEST_JOBSTEP_20150714_002881	Masters in Akmens Ridge are chasing the monster causing mayhem and disappearing. Defeat Netherbovine that appeared in Poslinkis Forest.
-QUEST_JOBSTEP_20150714_002882	Meet the Sorcerer Master in Forst of Prayers.
-QUEST_JOBSTEP_20150714_002883	The Sorcerer Master told you that you should not stay still since the monsters are going crazy these days. Go to Goddess' Ancient Gardenes and defeat Necrobenter.
+QUEST_JOBSTEP_20150714_002882	Meet the Sorcerer Master in Forest of Prayers.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Forest of Prayers -> Forest of Prayer

Also, remove the period. (Could you do a simple CTRL + F to change this along with the removal of the period, to see if there are any more lines ?)

File: QUEST_JOBSTEP.tsv [hunk 14]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
+QUEST_JOBSTEP_20150714_002872	Defeat Red Tree Ambulos at Apsimesti Crossroads.
+QUEST_JOBSTEP_20150714_002873	The Sadhu Master asked you to defeat the monsters that are threatening the villagers. Defeat Red Tree Ambulos from Apsimesti Crossroads.
 QUEST_JOBSTEP_20150714_002874	Talk to the Monk Master in the Goddess' Ancient Garden.
-QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants hardened oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
-QUEST_JOBSTEP_20150714_002876	go to the Sorcerer Master in Forest of Prayer.
-QUEST_JOBSTEP_20150714_002877	Rodolero Master told you that you are now a true Rodelero, but also told you that you can still improve. Defeat Colimencia at the garden of Green Family and find out what's lacking in you.
+QUEST_JOBSTEP_20150714_002875	The Tools Merchant wants Hardened Oil in exchange for information. Gather Hardened Oil from Puragis in Saknis Plains and give it to the Tools Merchant.
+QUEST_JOBSTEP_20150714_002876	Go to the Sorcerer Master in Forest of Prayer.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Remove the period.

File: QUEST_JOBSTEP.tsv [hunk 15]
```diff
@@ -2856,31 +2856,31 @@ QUEST_JOBSTEP_20150714_002855	Go to the Wugushi Master in Gateway to Great King
...
+QUEST_JOBSTEP_20150714_002868	Collect Sharp Canines from Netherbovine at Poslinkis Forest
+QUEST_JOBSTEP_20150714_002869	The Monk Master asked you to bring the proof of the determination. Defeat Netherbovine at Poslinkis Forest and bring Sharp Canines as a proof.
 QUEST_JOBSTEP_20150714_002870	Go to the Pardoner Master in Akmens Ridge.
-QUEST_JOBSTEP_20150714_002871	The Pardoner Master will let you skip the advancement assignment, but asked you to retrieve the money of lost indulgence. Find the lost money at Aqueduct Bridge Area.
-QUEST_JOBSTEP_20150714_002872	Defeat Red Triambulos at Apsimesti Crossroads.
-QUEST_JOBSTEP_20150714_002873	The Sadhu Master asked you to defeat the monsters that are threatening the villagers. Defeat Red Triambulos from Apsimesti Crossroads.
+QUEST_JOBSTEP_20150714_002871	The Pardoner Master will let you skip the advancement assignment, but asked you to retrieve the Lost Indulgence Offerings. Find the Lost Indulgence Offering at Aqueduct Bridge Area.
+QUEST_JOBSTEP_20150714_002872	Defeat Red Tree Ambulos at Apsimesti Crossroads.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Remove the period.

File: QUEST_JOBSTEP.tsv [hunk 16]
```diff
@@ -2911,8 +2911,8 @@ QUEST_JOBSTEP_20150717_002910	Give the piece of father's clothes to Tools Mercha
 QUEST_JOBSTEP_20150717_002911	You weren't able to find the merchant's father but you found some clothing that seems have belonged to him. Give it to the Tools Merchant.
 QUEST_JOBSTEP_20150717_002912	Received special bait feed from the Tools Merchant. Use this to lure the cat in Klaipeda.
 QUEST_JOBSTEP_20150717_002913	Meet Fedimian Tools Merchant
-QUEST_JOBSTEP_20150717_002914	In exchange for hardened oil, the Tools Merchant of Fedimian gold you that the monster named Mothstem has the Fire Pot. Ask the merchant where you can meet Mothstem.
-QUEST_JOBSTEP_20150717_002915	The Tools Merchant says he has met Mothstem in Fedimian Suburbs. Defeat the Mothstem in Fedimian Suburbs and get the Fire Pot.
+QUEST_JOBSTEP_20150717_002914	In exchange for Hardened Oil, the Tools Merchant of Fedimian told you that a monster named Mothstem has a Fire Jar. Ask the merchant where you can meet Mothstem.
+QUEST_JOBSTEP_20150717_002915	The Tools Merchant says he has met Mothstem in Fedimian Suburbs. Defeat the Mothstem in Fedimian Suburbs and get a Mostem Fire Jar.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: Mostem Fire Jar -> Mothstem's Fire Jar (this should be changed in ITEM as well)

File: QUEST_JOBSTEP.tsv [hunk 17]
```diff
@@ -3027,15 +3027,15 @@ QUEST_JOBSTEP_20150729_003026	{memo X}Tell him to go there himself
...
-QUEST_JOBSTEP_20150729_003034	As a return for obtaining cooperation from the Peltasta Master in Klaipeda, please obtain coarse tree barks from Yognomes in Tenet Chapel B1 and give them to him.
-QUEST_JOBSTEP_20150729_003035	Collect black needles from Glizardons in Tenet Chapel B1.
-QUEST_JOBSTEP_20150729_003036	The Peltasta Master gave you one mission. Obtain black nails from Glizardons in Tenet Chapel B1.
+QUEST_JOBSTEP_20150729_003030	Collect Desmodus' Tail Pins at Tenet Chapel B1
+QUEST_JOBSTEP_20150729_003031	The Psychokino Master told you that he is interested in this field and he asked you to become his assistant. Please obtain Desmodus' Tail Pins in Tenet Chapel B1 that are needed by the Psychokino Master.
+QUEST_JOBSTEP_20150729_003032	You've obtained enough Desmodus' Tail Pins. Return to the Psychokino Master and give them to him.
+QUEST_JOBSTEP_20150729_003033	Obtain Coarse Barks from Yognome in Tenet Chapel B1.
+QUEST_JOBSTEP_20150729_003034	As a condition for requesting cooperation from the Peltasta Master in Klaipeda, obtain Coarse Barks from Yognomes in Tenet Chapel B1 and give them to him.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: him -> her

The Peltasta Master is a female.

File: QUEST_JOBSTEP.tsv [hunk 18]
```diff
@@ -3027,15 +3027,15 @@ QUEST_JOBSTEP_20150729_003026	{memo X}Tell him to go there himself
...
+QUEST_JOBSTEP_20150729_003035	Collect black nails from Glizardons in Tenet Chapel B1.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: black nails -> Black Nails

File: QUEST_JOBSTEP.tsv [hunk 19]
```diff
@@ -3027,15 +3027,15 @@ QUEST_JOBSTEP_20150729_003026	{memo X}Tell him to go there himself
...
+QUEST_JOBSTEP_20150729_003036	The Peltasta Master gave you a mission. Obtain black nails from Glizardons in Tenet Chapel B1.
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: black nails -> Black Nails

File: QUEST_JOBSTEP.tsv [hunk 20]
```diff
@@ -3103,7 +3103,7 @@ QUEST_JOBSTEP_20150803_003102	If you could help my task, I think the investigati
 QUEST_JOBSTEP_20150803_003103	I am glad that Vaidotas asked me a favor.{nl}Return with this.
 QUEST_JOBSTEP_20150803_003104	Good.{nl}I need the orb that records the memories of the past from the Chronomancer Master.{nl}He is the head of our Winterspoon Family so I've requested it to him in advance. Tell him that I've sent you.
 QUEST_JOBSTEP_20150803_003105	Lucid... Actually have you met the Chronomancer Master well?
-QUEST_JOBSTEP_20150803_003106	The orb reads the memories of the possessor and records them.{nl}Explore all areas in the 1st and 2nd districts of the Crystal Mine with this.{nl}
+QUEST_JOBSTEP_20150803_003106	The orb reads the memories of the possessor and records them.{nl}Explore all areas in District 1 and 2 in the Crystal Mine with this.{nl}
```
Comment (Sunaries) [inline QUEST_JOBSTEP.tsv]: The orb reads the memories -> The orb reads the past memories

**Comment by Sunaries** [comment]
:+1: good to merge

**Comment by imcgames** [comment]
:+1:

**Pull request merged**